django/views/defaults.py

```diff
@@ -76,7 +80,12 @@ def server_error(request, template_name=ERROR_500_TEMPLATE_NAME):
         if template_name != ERROR_500_TEMPLATE_NAME:
             # Reraise if it's a missing custom template.
             raise
-        return HttpResponseServerError('<h1>Server Error (500)</h1>', content_type='text/html')
+        return HttpResponseServerError(
+            HTML_DOCUMENT.format(
```

Comment:
This can be single-lined, e.g.
```python
return HttpResponseServerError(
    ERROR_PAGE_TEMPLATE % {'title': 'Server Error (500)', 'details': ''},
    content_type='text/html',
)
```
The same in `bad_request()` and `permission_denied()`.